src/parser/scalar.js

```diff
@@ -188,18 +188,17 @@ module.exports = {
       name = null;
       if (this.next().token === this.tok.T_STRING_VARNAME) {
         const varName = this.text();
-        name = this.node("variable");
```

Comment:
I need to start the variable node builder before the next statement in order to correctly locate its offset

Reply:
Oh, that explains why it was written that way! I reverted the change.